core-war/src/main/webapp/util/javaScript/lookV5/navigation.js

```diff
@@ -1424,4 +1423,10 @@ whenSilverpeasReady(function() {
   spLayout.getBody().getNavigation().addEventListener('changeselected', __scrollListener, '__id__navigation-part');
   spLayout.getFooter().addEventListener('pdcshow', __scrollListener, '__id__navigation-part');
   spLayout.getFooter().addEventListener('pdchide', __scrollListener, '__id__navigation-part');
-});
\ No newline at end of file
+});
+
+function isDOM(getElem){
```

Comment:
This function MUST be declared at same level as the `__scrollListener` one.
By this way, there is no impact with other functionalities as the method is scoped.

Reply:
ok c'est corrigé